docs/guides/using-organizations.md

```diff
@@ -0,0 +1,133 @@
+
+## Switch to the preview image and enable the experiment
+
+To try the latest organizations features, switch to a preview image in your Helm
+chart and enable the [experimental flag](../cli/server.md#--experiments).
+
+For example, with Kubernetes, set the following in your `values.yaml`:
+
+```yaml
+coderd:
+  image:
+    repo: ghcr.io/coder/coder-preview
+    tag: orgs-preview-aug-16
+  env:
+    - name: CODER_EXPERIMENTS
+      value: multi-organization
+```
+
+> See all
+> [preview images](https://github.com/coder/coder/pkgs/container/coder-preview)
+> in GitHub. Preview images prefixed with `main-` expire after a week.
+
+Then, upgrade your deployment:
+
+```sh
+helm upgrade coder coder-v2/coder -f values.yaml
+```
+
+## The default organization
+
+All Coder deployments start with one organization called `Default`.
+
+To edit the organization details, navigate to `Deployment -> Organizations` in
+the top bar:
+
+![Organizations Menu](../images/guides/using-organizations/deployment-organizations.png)
+
+From there, you can manage the name, icon, description, users, and groups:
+
+![Organization Settings](../images/guides/using-organizations/default-organization.png)
+
+## Guide: Your first organization
+
+### 1. Create the organization
+
+Within the sidebar, click `New organization` to create an organization. In this
+example, we'll create the `data-platform` org.
+
+![New Organization](../images/guides/using-organizations/new-organization.png)
+
+From there, let's deploy a provisioner and template for this organization.
+
+### 2. Deploy a provisioner
+
+[Provisioners](../admin/provisioners.md) are organization-scoped and are
+responsible for executing Terraform/OpenTofu to provision the infrastructure for
+workspaces and testing templates. Before creating templates, we must deploy at
+least one provisioner:
+
+using Coder CLI, run the following command to create a key that will be used to
+authenticate the provisioner:
+
+```sh
+coder provisioner keys create data-cluster-key --org data-platform
+Successfully created provisioner key data-cluster! Save this authentication token, it will not be shown again.
+
+< key omitted >
+```
+
+Next, on your desired platform, start the provisioner with the key. See our
+[provisioner documentation](../admin/provisioners.md) for details on running on
+additional platforms (e.g. Kubernetes). In this example, we'll start it directly
+with the Coder CLI on a host with Docker:
+
+```sh
+export CODER_URL=https://<your-coder-url>
+export CODER_PROVISIONER_DAEMON_KEY=<key>
+coder provisionerd start --org <org-name>
+```
+
+### 3. Create a template
+
+Once you've started a provisioner, you can create a template. You'll notice the
+"Create Template" screen now has an organization dropdown:
+
+![Template Org Picker](../images/guides/using-organizations/template-org-picker.png)
+
+### 4. Create a workspace
+
+Now, users in the data platform organization will see the templates related to
```

Comment:
Looks good as-is, but I'd propose an alternate flow:

I'd swap the "create a workspace" step with the "add members" step. 

This flow makes more sense imo 
1. Create a template
2. allocate which users you want to use that template
3. Those users can create a workspace like this...

Reply:
makes sense! done